### ISSUE TITLE
Support subscribing to newly published or opened events

### ISSUE THREAD

**Issue by henrikhorluck** [heddendorp/tumi-app#43]
Why? It is close to impossible to know when new events are available without checking regularly.

Email reminders are probably the most cross-platform compatible, but using e.g. a Telegram newsletter, or https://developer.mozilla.org/en-US/docs/Web/API/notification, are also possibilities.

**Comment by euberdeveloper** [comment]
Hi Henrik, I made something similar to your issue https://github.com/euberdeveloper/tumi-app-bot

**Comment by heddendorp** [comment]
Also @henrikhorluck there will soon be registration opening dates.
Meaning:
Events will be published before the registration opens, allowing people to better plan their attendance and then sign up once registration opens.

**Comment by henrikhorluck** [comment]
> Also @henrikhorluck there will soon be registration opening dates.
> Meaning:
> Events will be published before the registration opens, allowing people to better plan their attendance and then sign up once registration opens.

Looking forward to it! I am assuming you are meaning signup time, and not just date :D

**Comment by henrikhorluck** [comment]
btw why isn't this project under https://github.com/ESNGermany?

**Comment by heddendorp** [comment]
Of course, it won't only be on midnight, I'm used to `Date` including the time, I guess.
I wouldn't think that this is a german matter 🤔 Why would you expect the project in the ESNGermany organisation.

_Also:_ this project ist older than the Github organisation of ESNGermany 😉
